src/InputField/index.js

```diff
@@ -166,9 +172,12 @@ export const Prefix = styled(({ children, className }) => (
   z-index: 3;
 
   & > svg {
+    color: ${({ theme }) => theme.orbit.colorIconInput};
+  }
+
+  & * svg {
```

Comment:
When `svg` is in `Prefix` directly, it won't change the size properly.

Reply:
Can you elaborate here?

Reply:
Check the svg size in Prefix in this story: https://orbit-components-new-error-forms.surge.sh/?path=/story/inputfield--with-error